vertical-pod-autoscaler/pkg/recommender/model/cluster.go

```diff
@@ -38,6 +38,8 @@ const (
 // events (container OOMs).
 // All input to the VPA Recommender algorithm lives in this structure.
 type ClusterState struct {
+	// Denote the recommender name
+	RecommenderName string
```

Comment:
As commented above I don't yet understand why we need to store this dynamically in the default recommender.

Reply:
Please split this change (adding `RecommenderName` field) to a separate PR.

It will make this PR easier to review. It will also make it easier to see how we'd use the field in alternative recommenders

Reply:
Please move this change (adding `RecommenderName` field) to a separate PR.

It will make this PR easier to review. It will also make it easier to see how we'd use RecommenderName field